src/help/plugin.ts

```diff
@@ -140,6 +144,15 @@ function activateHelpHandler(app: JupyterLab, mainMenu: IMainMenu, palette: ICom
     category: 'Help'
   }));
 
+  let openLegacyNotebookId = 'legacy-notebook:open';
+  app.commands.addCommand(openLegacyNotebookId, {
+    label: 'Legacy Jupyter Notebook',
```

Comment:
I tend to like "classic" rather than "legacy"...

Would also start the command with a verb: "Open Classic Notebook"